pandera/backends/pyspark/container.py

```diff
@@ -357,7 +357,7 @@ def strict_filter_columns(
                     )
 
         if schema.strict == "filter":
-            check_obj = check_obj.drop(*filter_out_columns)
+            check_obj = check_obj.drop(columns=filter_out_columns)
```

Comment:
looks like tests are failing due to this line: https://github.com/unionai-oss/pandera/actions/runs/5786938731/job/15682750086?pr=1291